tests/integration/test_hyper_transformer.py

```diff
@@ -228,7 +228,7 @@ def test_dtype_category():
 
```

Comment:
It seems like the `test_dtype_category` above this line fails because the `dtype` is lost during the transform/reverse cycle.
A similar inconsistency was reported for the Boolean Transformer, so I'm opening an issue to take care of this in the `BaseTransformer`.

@fealho can you flag this as an [expected failure using pytest](https://docs.pytest.org/en/latest/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail) in the meantime?